__tests__/index.test.ts

```diff
@@ -159,6 +161,27 @@ describe("local: create local table", () => {
     expect(tables.B.get()).toHaveProperty("bool", true);
     expect(tables.B.get()).toHaveProperty("array", [defaultEntity]);
   });
+
+  it("should be able to retrieve the state from local storage for a persisted table", async () => {
+    const world = createWorld();
+    const tables = {
+      A: createLocalBigIntTable(world, { id: "A", persist: true }),
+      B: createLocalTable(world, { bool: Type.Boolean, array: Type.EntityArray }, { id: "B", persist: true }),
+    };
+
+    tables.A.set({ value: BigInt(1) });
+    tables.B.set({ bool: true, array: [defaultEntity] });
+
+    const nextWorld = createWorld();
+    const nextTables = {
+      A: createLocalBigIntTable(nextWorld, { id: "A", persist: true }),
+      B: createLocalTable(nextWorld, { bool: Type.Boolean, array: Type.EntityArray }, { id: "B", persist: true }),
+    };
+
+    expect(nextTables.A.get()).toHaveProperty("value", BigInt(1));
+    expect(nextTables.B.get()).toHaveProperty("bool", true);
+    expect(nextTables.B.get()).toHaveProperty("array", [defaultEntity]);
+  });
 });
```

Comment:
Are there not more things to test here?
- Get properties,
- Set properties,
- Update properties